src/ansible_navigator/configuration_subsystem/utils.py

```diff
@@ -0,0 +1,22 @@
+"""Utilities related to the configuration subsystem."""
```

Comment:
I wonder if this module could be submitted separately to make the life of the reviewers easier.

Reply:
Also, we have to come up with a better name than a meaningless "utils". Utils doing what? It's not any better than "helpers", or "functions", or "stuff".
It looks like it's maybe a `settings_utils` or something like that.

Reply:
? It's name is `ansible_navigator.configuration_subsystem.utils` which I think indicates it is utils related to the configuration subsystem....

Am I missing something here?